SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -72,6 +75,17 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 				}
 
 				bWroteSomething = true;
+#if USE_NETWORK_PROFILER
+				/**
+				 *  a good proxy for how many bits are being sent for a propery. Reasons for why it might not be fully accurate:
+						• the serialized size of a message is just the body contents. Typically something will send the message with the length prefixed, which might be varint encoded, and you pushing the size over some size can cause the encoding of the length be bigger
```

Comment:
just noticed that you've got bullet chars in the comment... should probably switch that to hyphens in case anyone ever opens this in an editor that doesn't support properly utf8

Reply:
Done